apps/docs/gatsby-node.js

```diff
@@ -28,3 +30,27 @@ exports.onCreateWebpackConfig = ({actions, loaders, plugins, getConfig}) => {
 
   actions.replaceWebpackConfig(config)
 }
+
+exports.onPostBootstrap = async ({reporter}) => {
+  // eslint-disable-next-line i18n-text/no-en
+  reporter.info('Running post-bootstrap tasks...')
+
+  if (process.env.NODE_ENV === 'production') {
+    // eslint-disable-next-line i18n-text/no-en
+    reporter.info('No post-bootstrap tasks to run in production environment.')
+    return
+  }
+
+  const staticDir = path.resolve(__dirname, 'static', 'assets')
```

Comment:
nit, but could we call this `srcDir` and the other `outDir` please to make it immediately obvious what these are...